src/input.js

```diff
@@ -67,12 +65,6 @@ class Input extends Component {
     this.updateInputValue(event.target.value)
   }
 
-  handleBlur = () => {
-    if (!this.autocomplete.isMouseDown) {
-      this.autocomplete.reset()
-    }
-  }
-
   reset = () => {
     moveCursorToTheEnd(this._inputNode)
```

Comment:
Hah, yeah, maybe we should remove this call (and the method itself from utils because it's not used elsewhere).

Would you like to make that change @dashed?